src/make.js

```diff
@@ -0,0 +1,8 @@
+module.exports = function make(...args) {
+  const cache = args.slice();
+  return function f(...items) {
+    if (typeof items[0] === 'function') return cache.reduce(items[0]);
```

Comment:
Even though eslint allows not to specify braces for the single line conditional construction, most developers do it anyway since in large files it can be difficult to notice the condition without a block.

P.S. It's the point that influence the mark for this task, but nevertheless I highly recommend to use block structure like:
`if(SOME_CONDITION) {
    // your code, even if it`s a single line :)
}`

Reply:
Also, in your solution you expect the function to be the first parameter of one of the calls.
`if (typeof items[0] === 'function') return cache.reduce(items[0]);`
But in this case a calls, like:
`make(1, 2, 3, 4, 5, f)` or `make(1, 2)(3, 4, 5, f)` will not return the result. They will return the chain function instead.